content/post/release-of-1.4.0.md

```diff
@@ -0,0 +1,189 @@
+
+![Mention completion demo](/demos/3136/1.gif)
+
+"Is this the right username?" is no more. With our shiny new mention completion, you can now start typing an username or a name of someone on your Gitea instance, and results will start popping up as you type.
+
+_Thanks to **[@harryxu](https://github.com/harryxu)**_
+
+## Progress bar for issues with checkboxes ([#3171](https://github.com/go-gitea/gitea/pull/3171))
+
+![Progress bar demo](/demos/3171/1.png)
+
+Did you know that Gitea has, like GitHub, support for checkboxes in markdown? If you write a bulletpoint in the following format: `- [ ]` you will get a nice HTML textbox. And does this have to do with checkboxes? Everything, of course! The issue list will now show the percentage of checked items in the top issue post. So you always know how far you've gotten on the roadmap.
+
+_Thanks to **[@modmew8](https://github.com/modmew8)**_
+
+## Long commits now have an expandable body ([#2980](https://github.com/go-gitea/gitea/pull/2980))
+
+![Commit body expansion demo](/demos/2980/1.gif)
+
+Making a long commit? Now you can view the entire contents of it without changing the page; you will only need to press on the ellipsis button, similar to what you can see on GitHub and GitLab.
+
+_Thanks to **[@sondr3](https://github.com/sondr3)**_
+
+## Mark all notifications as read ([#3097](https://github.com/go-gitea/gitea/pull/3097))
+
+![Mark notifications as read demo](/demos/3097/1.png)
+
+Woah. You just came back from your holidays, and you should probably catch up on everything that happened at work... but should you? You can always sneakily dismiss all notifications. This tiny little button will also make that much easier for you ;).
+
+_Thanks to **[@svarmalov](https://github.com/svarmalov)**_
+
+## Write access for deploy keys ([#3225](https://github.com/go-gitea/gitea/pull/3225))
+
+![Screenshot showing a Read/Write deploy key and a Read deploy key](/demos/3225/1.png)
+
+In some cases, you may need to grant your server write access to the repository. Who knows, perhaps because of a tool that automatically generates and commits code? Either way, starting today, deploy keys can now be set to also have the ability to "write" to a repository. Power to the servers!
+
+_Thanks to **[@vtemian](https://github.com/vtemian)**_
+
+## Customize Gitea
+
+TODO: set of PRs that enable better customizability
+
+* https://github.com/go-gitea/gitea/pull/3051
+* https://github.com/go-gitea/gitea/pull/3286
+* https://github.com/go-gitea/gitea/issues/2115
+* https://github.com/go-gitea/gitea/pull/3345
+* https://docs.gitea.io/en-us/customizing-gitea/#customizing-gitea-pages
+
+TODO: quick guide on how to customize gitea, and tell users about the custom
+folder.
+
+# Other changes
+
+- **BREAKING:** if you used `GOGS_WORK_DIR` to change the working directory of Gitea, that won't work anymore - you'll need to use `GITEA_WORK_DIR`. ([#2946](https://github.com/go-gitea/gitea/pull/2946))
```

Comment:
I _very much_ think we should have _all_ breaking changes at the top. Otherwise people _will_ miss them (who reads changelogs anyway? :trollface: )

Reply:
wdym by at the top? at the top of the article? or is there some sarcasm I didn't catch? :P

Reply:
At the top of the article.